src/components/subtitlesync/subtitlesync.template.html

```diff
@@ -3,7 +3,7 @@
         <button type="button" is="paper-icon-button-light" class="subtitleSync-closeButton"><span class="material-icons close" aria-hidden="true"></span></button>
         <div class="subtitleSyncTextField" contenteditable="true" spellcheck="false">0s</div>
         <div class="sliderContainer subtitleSyncSliderContainer">
-            <input is="emby-slider" type="range" step=".1" min="0" max="100" value="50" class="subtitleSyncSlider" data-slider-keep-progress="true" />
+            <input is="emby-slider" type="range" step=".1" min="0" max="600" value="300" class="subtitleSyncSlider" data-slider-keep-progress="true" />
```

Comment:
Have you tried setting `min="-300" max="300"`? Iirc, `emby-slider` supports negative values. In that case, there will be less math.

Reply:
I'll try it out, if it works, I'll push it.

Reply:
It worked, I've pushed it.